larq_compute_engine/core/bgemm_kernels_arm.h

```diff
@@ -21,20 +21,6 @@ using compute_engine::core::TBitpacked;
 
 #if RUY_PLATFORM_NEON
 
-// Generic kNeon template when no types are specified
-template <typename DstScalar, typename MulParamsType>
-struct BgemmKernel<ruy::Path::kNeon, DstScalar, MulParamsType> {
-  ruy::Tuning tuning = Tuning::kAuto;
-  using LhsLayout = FixedKernelLayout<Order::kRowMajor, 1, 8>;
-  using RhsLayout = FixedKernelLayout<Order::kRowMajor, 1, 8>;
-  explicit BgemmKernel(ruy::Tuning tuning_) : tuning(tuning_) {}
-  void Run(const ruy::PMat<TBitpacked>& lhs, const ruy::PMat<TBitpacked>& rhs,
-           const MulParamsType& mul_params, int start_row, int start_col,
-           int end_row, int end_col, ruy::Mat<DstScalar>* dst) const {
-    TFLITE_DCHECK(false);
-  }
-};
```

Comment:
We've been able to remove this because the Aarch64/Arm32 kernel declarations below are now generic over `DstScalar`, which means we don't need to declare a `kNeon` path kernel without types.